raiden/api/validations/light_client_safe_operation.py

```diff
@@ -0,0 +1,34 @@
+from http import HTTPStatus
+
+import structlog
+import traceback
+
+from raiden.api.validations.api_error_builder import ApiErrorBuilder
+from raiden.api.validations.light_client_authorization import requires_api_key
+from raiden.exceptions import InsufficientFunds
+
+log = structlog.get_logger(__name__)
+
+
+def lc_safe_operation(func):
+    """
+        Executes an lc operation and never pull down the node, instead it always respond with an error code.
```

Comment:
```suggestion
        Executes an operation and never shuts down the node, instead it always responds with an error code.
```

Reply:
Shuts seems like a controlled scenario and this is for unhandled scenarios, maybe `throw the node down` or something similar sounds better.

Reply:
i'd say `crashes` if you want uncontrolled shutdown semantics. `throw down` means a lot of other different things :stuck_out_tongue:

Reply:
Sounds better will fix it